raiden/storage/migrations/v19_to_v20.py

```diff
@@ -4,15 +4,22 @@
 from eth_utils import to_canonical_address
 from gevent.pool import Pool
 
+from raiden import raiden_service  # pylint: disable=unused-import
```

Comment:
can you please just import `RaidenService` class directly here?

Reply:
Also, this can be wrapped in `if TYPE_CHECKING, import RaidenService` since we're keeping the type "quoted" in the function declarations